devito/core/autotuning.py

```diff
@@ -24,6 +29,7 @@ def autotune(operator, arguments, tunable, mode='basic'):
 
     # Squeeze dimensions to minimize auto-tuning time
     iterations = FindNodes(Iteration).visit(operator.body)
+    dim_mapper = {i.dim.name: i.dim for i in iterations}
```

Comment:
Yeah as I mentioned in my first comment, when I forgot to call the autotuner from `arguments`, all tests still passed so I am only pointing out that there needs to be a test to catch that.